spark_log_parser/parsing_models/application_model.py

```diff
@@ -201,14 +201,14 @@ def __init__(self, eventlogpath, bucket=None, stdoutpath=None, debug=False):  #
                     event_keys = spark_properties.keys()
 
                     # This if is specifically for databricks logs
-                    if spark_version := spark_properties["spark.databricks.clusterUsageTags.sparkVersion"]:
```

Comment:
rookie Python mistake - indexing a dict with a value that isn't present in the dict raises a `KeyError`. `get()` just returns `None`, which is what was intended